.circleci/config.yml

```diff
@@ -79,19 +79,27 @@ jobs:
       - run:
           name: Tag Docker Image
           command: |
-            cat docker-cache/docker-tags.txt \
-              | xargs -t -I % \
-                docker tag \
-                  "$DOCKER_REPOSITORY:$CIRCLE_SHA1" \
-                  "$DOCKER_REPOSITORY:%"
+            if [ -z "$CIRCLE_PR_USERNAME" ]; then \
+              cat docker-cache/docker-tags.txt \
+                | xargs -t -I % \
+                  docker tag \
+                    "$DOCKER_REPOSITORY:$CIRCLE_SHA1" \
+                    "$DOCKER_REPOSITORY:%"
+            else
+              echo "No tagging for forks"
+            fi
```

Comment:
@impactmass did you have a particular reason for adding the condition on the tagging step? I don't see any harm with the tags being present **if** we don't push. The tags would only exist in the local build. I'd prefer to keep conditionals to a single location if possible.

Reply:
My thought was that since they aren't getting pushed the tagging can be skipped. If it's okay to have them in the local builds, I can enable. There'll be a slight modification to the tagging (because docker tagging fails tags with the `/` character).

Reply:
Yep, good find! You found a bug that the acceptable branch name chars and Docker tag chars aren't the same set. Sound OK to handle that as a separate issue?

Reply:
Yes, sounds good to put that separate. I'll update this to remove the condition around tagging.